tests/ssl-opt.sh

```diff
@@ -1340,6 +1340,40 @@ run_test    "Renegotiation: server-initiated" \
             -s "=> renegotiate" \
             -s "write hello request"
 
+# Checks that no Signature Algorithm with SHA-1 gets negotiated. Negotiating SHA-1 would mean that
+# the server did not parse the Signature Algorithm extension. This test is valid only if an MD
+# algorithm stronger than SHA1 is enabled in config.h
+run_test    "Renegotiation: Signature Algorithms parsing, client-initiated" \
```

Comment:
Just in case you intend to correct the typo in the ChangeLog, can you replace `SHA1` by `SHA-1` here, too? If not, this can of course be ignored.